### PR TITLE
fix: rename setPublicationStrategy and getPublicationStrategy arguments.

### PR DESCRIPTION
**Pull request by fredmaiaarantes**
Rename setPublicationStrategy and getPublicationStrategy arguments.

Fix issue: [Bad naming for setPublicationStrategy variable (documentation fix probably) #12022](https://github.com/meteor/meteor/issues/12022)

### CODE REVIEW

File: packages/ddp-server/livedata_server.js [hunk 1]
```diff
@@ -1520,30 +1520,30 @@ Object.assign(Server.prototype, {
    * @summary Set publication strategy for the given publication. Publications strategies are available from `DDPServer.publicationStrategies`. You call this method from `Meteor.server`, like `Meteor.server.setPublicationStrategy()`
```
Comment (radekmie) [inline packages/ddp-server/livedata_server.js:1520]: Both JSDocs still suggest that it's set per publication, not collection.

**Review by radekmie — APPROVED** [review]

File: packages/ddp-server/livedata_server.js (hunk 1 above)
Comment (fredmaiaarantes) [reply packages/ddp-server/livedata_server.js]: Thanks! I just changed them as well.

**Comment by fredmaiaarantes** [comment]
The variable name initially was 'collectionName,' not 'publicationName.' Later, when docs were added, I believe the variable name was changed by mistake.

**Review by Grubba27 — APPROVED** [review]
Looks good

**Pull request merged**